charts/rasa-x/templates/rasa-deployments.yaml

```diff
@@ -31,6 +31,7 @@ spec:
       tolerations:
 {{ toYaml $.Values.rasa.tolerations | indent 6 }}
       {{- end }}
+      {{- include "initContainer.dbMigration" $ |  nindent 6 }}
```

Comment:
why?

Reply:
I use the init container for the rasa deployment because the liveness probe for the rasa-production and rasa-worker service doesn't pass as long as the rasa-x is not fully operational.  

To make sure that the services are not restarting and have a clear start, first, we check for a DB migration and we make sure that everything is ready to go!

Without the init container:
```
rasa-x-rasa-production-65c964c9df-6h88r   1/1     Running   1          2m48s
rasa-x-rasa-worker-5d766bf7c4-5tpkr       1/1     Running   1          2m48s
```

with the init container:
```
rasa-x-rasa-production-678b9bb5f8-dntpm        1/1     Running            0          18m
rasa-x-rasa-worker-86b6498c6c-nqsxt            1/1     Running            0          18m
```

Reply:
Makes sense, but it's stupid, isn't it? I think it would be better if Rasa Open Source starts independent of what Rasa X. does and pulls the config later. Wdyt? 

We should also add your comment as comment to the deployment file to make it clear why we need the init container there.

Reply:
> Makes sense, but it's stupid, isn't it? I think it would be better if Rasa Open Source starts independent of what Rasa X. does and pulls the config later. Wdyt?

I see your point and I agree with you that it'd be better to start Rasa OS independently.

From the perspective of a user who uses the rasa-x helm chart first time, currently, after deploy, I see two services that ware restarted. The first reaction will be to check the logs of the services, but the logs are empty, zero information on why the services ware restarted.

I think if we can save someone from scratching in the head and wondering what happened, by simply adding the init container we should that.